java-frontend/src/main/java/org/sonar/java/model/JParser.java

```diff
@@ -844,6 +844,8 @@ private EnumConstantTreeImpl processEnumConstantDeclaration(EnumConstantDeclarat
       ),
       separatorToken
     );
+    t.variableBinding = e.resolveVariable();
```

Comment:
@m-g-sonar thank you for spotting this 👍however `declaration(t.variableBinding, t)` is missing in this change :wink: I added both into master in 482a3f933fac6dc3c1034763d4476089fb16a69b

Reply:
I have seen the fix on master :+1: .
I'll rebase the branch.